iceoryx_posh/include/iceoryx_posh/internal/popo/ports/publisher_port_user.hpp

```diff
@@ -56,6 +56,11 @@ class PublisherPortUser : public BasePort
     /// @param[in] chunkHeader, pointer to the ChunkHeader to free
     void freeChunk(mepoo::ChunkHeader* const chunkHeader) noexcept;
 
+    /// @todo decide for name but have the same interface on publisher and receiver side
+    /// @brief Free an allocated chunk without sending it
+    /// @param[in] chunkHeader, pointer to the ChunkHeader to free
+    void releaseChunk(mepoo::ChunkHeader* const chunkHeader) noexcept;
```

Comment:
Yes, I would favor releaseChunk since depending on who else holds references it might not be freed. I consider free to be the call where it is ensured that the shared memory can be reused afterwards, i.e. after the last reference is released.

Reply:
@MatthiasKillat my logic was exactly the same

Reply:
I removed freeChunk from the publisher port and we have only release for now. Only left to reflect that in the C API as well which will be in the next pull request concerned with renaming functions there.